internal/syncer/syncer.go

```diff
@@ -102,15 +102,15 @@ func (w *worker) exec(ctx context.Context, id string) {
 					continue
 				}
 
-				w.logger.Err(err).Msgf("error dequeuing job: %v", err)
+				w.logger.Warn().AnErr("Error", err).Msgf("error dequeuing job: %v", err)
 				continue
 			}
 
 			w.loggerForJob(j).Info().Msg("dequeued job")
 
 			if err := w.handle(ctx, j); err != nil {
 				if !errors.Is(err, context.Canceled) {
-					w.logger.Err(err).Msgf("error handling job: %v", j)
+					w.logger.Warn().AnErr("Error", err).Msgf("error handling job: %v", j)
```

Comment:
```suggestion
					w.logger.Warn().AnErr("error", err).Msgf("error handling job: %v", j)
```

This is probably the big one. This will absorb all the errors returned by a syncer and just warn on it.